communication.md

```diff
@@ -12,32 +12,43 @@ The SDK **MUST** use the [Telemetry ingest APIs](https://docs.newrelic.com/docs/
 
 ### User Agent
 
-All SDKs **MUST** send the following `User-Agent` header by default:
-`NewRelic-<language>-TelemetrySDK/<version>`
+The `User-Agent` header field is used to preform analytics on requests received by New Relic.
```

Comment:
Thanks for making the motivation clear here! 💯

Reply:
preform -> perform